voluncheer/profiles/test_password_reset.py

```diff
@@ -0,0 +1,100 @@
+from django.test import TestCase
+from django.urls import reverse
+from profiles.models import User
+from django.core import mail
+from django.utils.http import urlsafe_base64_encode
+from django.utils.encoding import force_bytes
+from django.contrib.auth.tokens import default_token_generator
+
+
+class PasswordResetTest(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(
+            email="testuser@example.com",
+            password="password",
+            is_active="True",
+            type="2",
+        )
+        self.test_user = User.objects.get(email="testuser@example.com")
+        self.test_user.is_active = True
+        self.test_user.save()
+        self.uidb64 = urlsafe_base64_encode(force_bytes(self.test_user.pk))
+        self.token = default_token_generator.make_token(self.test_user)
+
+    def test_password_reset_form_view(self):
+        response = self.client.get(reverse("password_reset"))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(
+            response, "registration/password_reset_form.html"
+        )  # noqa E501
```

Comment:
These lint and format exceptions should only be used as a last resort. I don't think this one is even necessary, is it?

Reply:
For some reason, until I used it every time I tried to restructure it on my own, after saving the code would go back to the original format I had. Oddly, using the exception restructured the code for me...